docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -65,6 +65,24 @@ def downstream_asset(upstream_asset):
 
 The [explicit dependencies](#explicit-dependencies) example covers an alternative way to specify asset dependencies without needing to match argument names to upstream asset names.
 
+### Graph-backed assets
+
+[Basic software-defined assets](#a-basic-software-defined-asset) can only produce one data artifact. If your use case requires an asset that produces more than one data artifact, you can use graph-backed assets.
```

Comment:
A couple comments:
* Graph-backed assets are still useful in some cases that don't involve multiple data assets.  Not convinced my description is 100%, but here's how I tried to explain it in the migration/upgrade guide: https://elementl.quip.com/BHJUAq5kVfZV/Upgrading-to-software-defined-assets.
* "artifact" is a term that we don't yet use elsewhere - would using just "asset" work here?

Reply:
Re: `artifact` - that's fine! I thought we were since I've seen it floating around. I'll take a look at the doc too and make some changes.